### PR TITLE
Updates v4

### PR DESCRIPTION
**Pull request by alibazlamit**
Add the option to provide image alias as `profitbricks-image` parameter
Minor changes to test
Add govendor dependency management

### CODE REVIEW

**Comment by alibazlamit** [comment]
@jasminSPC could you please take a look at this PR.
Thanks.

File: profitbricks.go [hunk 1]
```diff
@@ -302,7 +308,7 @@ func (d *Driver) Create() error {
 	}
 
 	nic := profitbricks.Nic{
-		Properties: profitbricks.NicProperties{
+		Properties: &profitbricks.NicProperties{
```
Comment (jasminSPC) [inline profitbricks.go]: why `NicProperties` are a pointer? This is probably related to the SDK.

File: profitbricks_test.go [hunk 2]
```diff
@@ -49,19 +50,24 @@ func getTestStorePath() (string, error) {
...
-			"swarm-host":             "2",
-			"swarm-discovery":        "3",
+			"profitbricks-endpoint":                 "https://api.profitbricks.com/cloudapi/v4",
+			"profitbricks-username":                 os.Getenv("PROFITBRICKS_USERNAME"),
+			"profitbricks-password":                 os.Getenv("PROFITBRICKS_PASSWORD"),
+			"profitbricks-disk-type":                "HDD",
+			"profitbricks-disk-size":                5,
+			"profitbricks-cpu-family":               "AMD_OPTERON",
+			"profitbricks-image":                    "Ubuntu-16.04",
+			"profitbricks-image-alias":              "Ubuntu:16.04",
```
Comment (jasminSPC) [inline profitbricks_test.go]: Since we are using only `--profitbricks-image` remove `profitbricks-image-alias`. Unless I missed something

Comment (alibazlamit) [inline profitbricks_test.go]: Removed

File: profitbricks.go (hunk 1 above)
Comment (alibazlamit) [inline profitbricks.go]: Yes it is related to the SDK.

**Comment by jasminSPC** [comment]
Once we merge the changes please build binaries using `make release` tag the release and upload binaries.

**Pull request merged**